src/listing/index.js

```diff
@@ -2,10 +2,13 @@ import { createTable } from 'webcharts';
 import callbacks from './callbacks/index';
 
 export default function listing(clinicalTimelines) {
-    const listing = createTable(clinicalTimelines.element, {
-        cols: clinicalTimelines.config.details.map(d => d.value_col),
-        headers: clinicalTimelines.config.details.map(d => d.label)
-    });
+    const listing = createTable(
+        clinicalTimelines.element,
+        clinicalTimelines.config.listingConfig || {
```

Comment:
Probably need to fine-tune this a bit so that the columns/labels are included in the custom config when provided.